gnn/automl.py

```diff
@@ -0,0 +1,69 @@
+from args import parse_args
+from train import train
+from eval import eval
+
+from recommenders.models.deeprec.deeprec_utils import prepare_hparams
+from recommenders.models.deeprec.DataModel.ImplicitCF import ImplicitCF
+from recommenders.datasets import movielens
+from recommenders.datasets.python_splitters import python_stratified_split
+
+import pandas as pd
+import numpy as np
+
+import pprint
+import itertools
+
+
+if __name__ == '__main__':
+    
+    args = parse_args()
+
+    params = prepare_hparams(
+        yaml_file=args.hparams_yaml_file,
+        load_epoch=args.load_epoch
+    )
+
+    dataset = movielens.load_pandas_df(size=params.dataset_size)
+
+    num_seeds = 10
+    original_seed = params.seed
+    all_metrics = {}
+    for k in params.metrics:
+        all_metrics[k] = []
+        all_metrics[k+'_std'] = []
+    all_metrics['modelname'] = []
+
+    for neighbor_aggregator in ['degree_norm', 'attention']:
+        for info_updater in ['direct', 'linear']:
```

Comment:
should be `for info_updater in ['direct', 'single_linear', 'multi_linear']` i think